recipe/meta.yaml

```
@@ -1,25 +1,26 @@
 {% set name = "starlette-context" %}
-{% set version = "0.3.5" %}
+{% set version = "0.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/starlette_context-{{ version }}.tar.gz
-  sha256: e6b9f905823860e9e36c013dbfcf770562f3b88bec21cb861fef2e0bd0615697
+  sha256: 3242417c9354c067a4ac5009aff762dc0b322074216f664825d5d127108553be
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [py<39]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - poetry-core
   run:
-    - python >=3.7
+    - python
     - starlette
```

```suggestion
    - starlette >=0.27.0
```
https://github.com/tomwojcik/starlette-context/blob/v0.4.0/pyproject.toml#L30C14-L30C22